flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/DefaultScheduler.java

```diff
@@ -401,12 +401,17 @@ private static void propagateIfNonNull(final Throwable throwable) {
 					.registerProducedPartitions(logicalSlot.getTaskManagerLocation(), sendScheduleOrUpdateConsumerMessage);
 				executionVertex.tryAssignResource(logicalSlot);
 			} else {
-				handleTaskFailure(executionVertexId, maybeWrapWithNoResourceAvailableException(throwable));
+				onTaskDeploymentFailure(executionVertexId, maybeWrapWithNoResourceAvailableException(throwable));
 			}
 			return null;
 		};
 	}
 
+	private void onTaskDeploymentFailure(final ExecutionVertexID executionVertexId, final Throwable error) {
```

Comment:
I would prefer `handleTaskDeploymentFailure` to be consistent.

Reply:
done.